gcp/README.md

```diff
@@ -77,3 +77,43 @@ This happens when trying to enable an API that is already enabled. This shouldn'
 ```
 rake xk"[gcloud services disable container.googleapis.com]"
 ```
+
+### Restoring CouchDB data
+
+We are considering number of probable failure scenarios for our GCP infrastructure:
+
+1. **Data corruption on a single CouchDB replica**
+
+In this scenario we rely on CouchDB ability to recover from loss of one or more replicas (our current production CouchDB settings allow us to lose up to 2 random nodes and still keep data integrity). The best course of action as follows:
+
+* Make sure that you figured affected CouchDB pod properly
+* There is a PVC object, associated with affected CouchDB pod. Let's say affected pod is `couchdb-couchdb-1`, then corresponding PVC is `database-storage-couchdb-couchdb-1`, located in the same namespace.
+* Delete associated PVC and then affected pod. For our example case:
+   * `kubectl --namespace gpii delete pvc database-storage-couchdb-couchdb-1`
+   * `kubectl --namespace gpii delete pod couchdb-couchdb-1`
+* After target pod is terminated, Persistent Disk that was mounted into it thru corresponding PVC will be destroyed as well.
+* When new pod is created to replace deleted one, corresponding PVC will be created as well, and, thru it, new PV object for new GCE PD.
+* Please note that you'll have to patch newly created PV, so `k8s-snapshots` can continue to make its snapshots, for our example case:
+   * Get volume name from `kubectl --namespace gpii get pvc database-storage-couchdb-couchdb-1`.
+   * Patch PV with `kubectl patch pv PV_NAME_FROM_PREVIOUS_STEP -p '{"metadata": {"annotations": {"backup.kubernetes.io/deltas": "ENV_DELTAS"}}}'`
```

Comment:
Hmm, this is kind of rough. Can we instead run `rake` again (or maybe `rake deploy_module[couchdb]`) and let the terraform code handle this for us?

Reply:
Cool. Looks like you can replace the top-level bullet at line 96 and all its children with "Run `rake deploy_module[couchdb]`".

Reply:
Done